runtime/src/installed_scheduler_pool.rs

```diff
@@ -167,7 +167,8 @@ pub trait InstalledScheduler: Send + Sync + Debug + 'static {
     /// having &mut.
     fn schedule_execution<'a>(
         &'a self,
-        transaction_with_index: &'a (&'a SanitizedTransaction, usize),
+        transaction: SanitizedTransaction,
+        index: usize,
```

Comment:
hehe, this is a remnant of never-shippted attempt of sneaking GAT into our code base... thanks for cleaning up.